src/main/kotlin/io/github/nomisrev/validation.kt

```diff
@@ -1,15 +1,10 @@
-@file:Suppress("TooManyFunctions")
+@file:Suppress("TooManyFunctions", "WildcardImport")
```

Comment:
No wildcard imports please 😅 I never use them as a best practice.

Reply:
I'm fine with that. I think had already changed my settings to not use them, but then spotless handed me an error message I couldn't make sense of. Will try again.

Reply:
Removed the wildcard imports.

Reply:
Can you use those changes?

Reply:
Thank you very much for getting back at me. I'll push the wildcard import changes this evening as soon as I have time.